src/services/authService.js

```diff
@@ -27,17 +27,39 @@ const AuthService = {
     return true;
   },
   isShelterAdmin() {
-    // TODO
-    return false;
+    const token = AuthService.getToken();
+    if (!token) {
+      return false;
+    }
+    // check JWT expiration
+    const tokenIsExpired = isExpired(token);
+    if (tokenIsExpired) {
+      return false;
+    }
+    // check JWT content
+    const tokenContent = decodeToken(token);
+    if (!tokenContent.hasOwnProperty('shelterID')) {
+      return false
+    }
+    return true;
   },
   async loginUser(credentials) {
-    return fetch(HOSTNAME + '/login', {
+    return fetch(HOSTNAME + '/users/login', {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json'
       },
       body: JSON.stringify(credentials)
     }).then((data) => data.json());
+  },
+  async loginShelter(credentials) {
```

Comment:
We can probably pass a parameter `userType`
`async loginUser(credentials, userType) {
    return fetch(HOSTNAME + '/{userType}/login', {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json'
      },
      body: JSON.stringify(credentials)
    }).then((data) => data.json());
  `

Reply:
I just tried this with passing parameters, and I think it's a little clearer when it's two separate functions, even if it is a little repetitive. Passing a string in the parameter as a parameter to determine the proper route is a little confusing and possibly prone to bugs. I could definitely be missing something though, so let me know what you think!

Reply:
I can see that it might be confusing. Thank you for testing it!